packages/minimongo/minimongo.js

```diff
@@ -221,14 +221,26 @@ LocalCollection.Cursor.prototype.fetch = function () {
  * @locus Anywhere
  * @returns {Number}
  */
-LocalCollection.Cursor.prototype.count = function () {
+LocalCollection.Cursor.prototype.count = function (applySkipLimit) {
   var self = this;
+  // Have to respect `applySkipLimit`, so we remove limit & skip if `applySkipLimit`
```

Comment:
Yeah, after chatting with @glasser I agree that we should do the thing you're suggesting where we break backwards compatibility so that the client and server behave the same way.

But we should give a smooth upgrade path for users instead of hoping that they read the upgrade guide.